packages/paste-core/components/table/src/Tr.tsx

```diff
@@ -1,26 +1,30 @@
 import * as React from 'react';
 import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
 import {TableContext} from './TableContext';
-import {TrProps, TrPropTypes} from './types';
+import type {TrProps} from './types';
+import {TrPropTypes} from './types';
 
-const Tr = React.forwardRef<HTMLTableRowElement, TrProps>(({verticalAlign = 'middle', ...props}, ref) => {
-  const {striped} = React.useContext(TableContext);
-  return (
-    <Box
-      {...striped}
-      {...safelySpreadBoxProps(props)}
-      ref={ref}
-      as="tr"
-      borderStyle="solid"
-      borderColor="colorBorderWeaker"
-      borderWidth="borderWidth0"
-      borderBottomWidth="borderWidth10"
-      verticalAlign={verticalAlign}
-      _even={{backgroundColor: striped ? 'colorBackgroundRowStriped' : 'transparent'}}
-      _last={{borderWidth: 'borderWidth0'}}
-    />
-  );
-});
+const Tr = React.forwardRef<HTMLTableRowElement, TrProps>(
+  ({element = 'TR', verticalAlign = 'middle', ...props}, ref) => {
+    const {striped} = React.useContext(TableContext);
+    return (
+      <Box
+        {...striped}
```

Comment:
Technically this is a boolean, so spreading it is jus spreading an empty object. Probably should remove this at some point, but leaving since it's out of scope.